db/migrate_cards/20151202112547_import_migrate_tags_to_topics.rb

```diff
@@ -0,0 +1,13 @@
+# -*- encoding : utf-8 -*-
+
+class ImportMigrateTagsToTopics < Card::Migration
+  def up
+    tags_card = Card.search type_id: Card::WikirateTagID
+    tags_card.each do |card|
+      card.type_id = Card::WikirateTopicID
+      puts "Updating #{card.name}'s type to topic"
+      card.save!
+    end
+    import_json "migrate_tags_to_topics.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.